engine/class_modules/paladin/sc_paladin.hpp

```diff
@@ -1145,11 +1145,12 @@ struct holy_power_consumer_t : public Base
     // Free sotr from vanq does not proc RP 2020-09-10
     if ( p -> talents.righteous_protector -> ok() && !ab::background )
     {
-      timespan_t reduction = timespan_t::from_seconds(
-        // Why do I need to divide this by 10? Just give me sec or milli, what is this??
-         -1.0 * p -> talents.righteous_protector -> effectN( 1 ).base_value()
-         * num_stacks / 10
-       );
+      // Why is this in deciseconds?
+      timespan_t reduction = -100_ms * p -> talents.righteous_protector -> effectN( 1 ).base_value();
```

Comment:
Can probabkly just keep using the old format here ``timespan from seconds(value / 10)`` since it's the format that the tooltip code applies. Stupidformattinginterpretation#1 or stupidformattinginterpretation#2 doesn't make much difference anyway.

You should also probably use cost() instead of num_stacks since that one is specific to crusade code (unless you expanded it somewhere else)

Reply:
Crusade, Righteous Protector, Relentless Inquisitor (the azerite version) (and I suppose fist of justice but NYI) all use the same logic of "only give value for holy power spent", so holy power reductions in general don't proc them. Hence using the same variable